src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.GetProcessMemoryInfo.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    [StructLayout(LayoutKind.Sequential)]
+    internal struct ProcessMemoryCounters
```

Comment:
We maintain casing and formatting of the platform identifiers in PInvokes. This should be `PROCESS_MEMORY_COUNTERS`